### PR TITLE
Task Track - Delete

### PR DESCRIPTION
**Pull request by leonidbogin**
image

### CODE REVIEW

File: DIMS-Core/DIMS-Core.BusinessLayer/Interfaces/ITaskTrackService.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿using DIMS_Core.BusinessLayer.Models.TaskTrack;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace DIMS_Core.BusinessLayer.Interfaces
+{
+    public interface ITaskTrackService
+    {
+        Task<IEnumerable<VTaskTrackModel>> GetAllForMember(int UserId);
```
Comment (SashaDrapun) [inline DIMS-Core/DIMS-Core.BusinessLayer/Interfaces/ITaskTrackService.cs:9]: Camelcase for method params.

File: DIMS-Core/DIMS-Core.BusinessLayer/Interfaces/IUserTaskService.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using DIMS_Core.BusinessLayer.Models.UserTask;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace DIMS_Core.BusinessLayer.Interfaces
+{
+    public interface IUserTaskService
+    {
+        Task<IEnumerable<UserTaskModel>> GetAllForMember(int UserId);
```
Comment (SashaDrapun) [inline DIMS-Core/DIMS-Core.BusinessLayer/Interfaces/IUserTaskService.cs:9]: Camelcase for method params.

File: DIMS-Core/DIMS-Core/Controllers/TaskTrackController.cs [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+namespace DIMS_Core.Controllers
+{
+    [Route("task-tracks")]
+    public class TaskTrackController : Controller
+    {
+        private readonly ITaskTrackService taskTrackService;
+        private readonly IUserTaskService userTaskService;
+        private readonly IMapper mapper;
+
+        public TaskTrackController(ITaskTrackService taskTrackService, IUserTaskService userTaskService, IMapper mapper)
```
Comment (SashaDrapun) [inline DIMS-Core/DIMS-Core/Controllers/TaskTrackController.cs:20]: Split parameters into several lines if the number of parameters passed is more than 2.

**Review by SashaDrapun — COMMENTED** [review]
Good job! I did some comments. Look them through and fix.

File: DIMS-Core/DIMS-Core.BusinessLayer/Interfaces/ITaskTrackService.cs (hunk 1 above)
Comment (leonidbogin) [reply DIMS-Core/DIMS-Core.BusinessLayer/Interfaces/ITaskTrackService.cs]: Task<IEnumerable<VTaskTrackModel>> GetAllByUserId(int userId);

File: DIMS-Core/DIMS-Core.BusinessLayer/Interfaces/IUserTaskService.cs (hunk 2 above)
Comment (leonidbogin) [reply DIMS-Core/DIMS-Core.BusinessLayer/Interfaces/IUserTaskService.cs]:
```suggestion
        Task<IEnumerable<UserTaskModel>> GetAllByUserId(int userId);

**Review by PushMeNow — APPROVED** [review]
Good job

**Pull request merged**